deck/models.py

```diff
@@ -243,6 +244,19 @@ def disapprove(self):
         self.is_approved = False
         self.save()
 
+    @property
+    def coauthors_names(self):
+        coauthors = self.coauthors.all().values('username')
+        if not coauthors:
+            return _("None")
+        return ", ".join(map(lambda x: x['username'], coauthors))
+
+    def get_authors_email(self):
```

Comment:
`get_authors_emails` as a plural looks better since it returns a list of emails.